### ISSUE TITLE
Bad triangle in first created solid

### ISSUE THREAD

**Issue by 2020karaulov2020** [wootguy/bspguy#39]
https://github.com/wootguy/bspguy/pull/37

![109385566-d2996e00-7905-11eb-93ca-f6739b165426 1](https://user-images.githubusercontent.com/62130676/109424065-30f04a80-79f3-11eb-92b7-c0e07199fc0c.png)

**Comment by wootguy** [comment]
Does this happen only with the `-createnew` code? I don't see this when adding a new solid to an existing map.

**Comment by 2020karaulov2020** [comment]
Yes If you create empty bsp(), and add solid. You lost one triangle, in game also this triangle not visible.
